schema/node.go

```diff
@@ -33,6 +33,7 @@ type Node struct {
 	APY                    decimal.Decimal        `json:"apy"`
 	ActiveScore            decimal.Decimal        `json:"activeScore"`
 	ReliabilityScore       decimal.Decimal        `json:"reliabilityScore"`
+	Type                   string                 `json:"type"`
```

Comment:
'Type' field has made the 'Alpha' field redundant.

Reply:
@brucexc 'Alpha' field is set by the smart contract and depends on timestamp. These two fields have different meanings.